apps/system/style/modal_dialog/modal_dialog.css

```diff
@@ -85,6 +85,7 @@
 
 #modal-dialog-select-one h3 {
   font-size: 1.8rem;
+  line-height: 2rem;
 }
 
 #modal-dialog-select-one ul {
```

Comment:
It's not obvious to me that this is an improvement.

If you hover on the `h3` in the inspector, the vertical guide shows that the first pixel of button text is aligned with that of the `h3`.

For the `line-height`, what's your reasoning that these particular buttons needs a different line height than the default from the `[role="dialog"] button` block?